### PR TITLE
DOR-89 Frotend > Crear Formulario de registro de libro

### PR DESCRIPTION
**Pull request by DodanimJR**
https://isc-ads.atlassian.net/browse/DOR-89

### CODE REVIEW

**Comment by wmartzh** [comment]
If there an error on data form does't catch it 

image

**Comment by wmartzh** [comment]
Picture should be optional
image

**Comment by wmartzh** [comment]
Form should reset after success message

image

File: src/services/book.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    const book = {
+      title: document.getElementById('BookTitle').value,
+      author: document.getElementById('BookAuthor').value,
+      isbn: document.getElementById('BookISBN').value,
+      category: document.getElementById('BookCategory').innerHTML.toString(),
+      year: document.getElementById('BookYear').getAttribute('value'),
+      picture: document.getElementById('BookPicture').value,
+      editorial: document.getElementById('BookEditorial').value,
+      language: document.getElementById('BookLanguage').innerHTML.toString()
+    };
```
Comment (wmartinezh) [inline src/services/book.js]: Use react state to capture data, all this function should on the form component, only backend request shold be called from outsied the component

File: src/pages/books/RegisterBook.jsx [hunk 2]
```diff
@@ -1,7 +1,105 @@
...
+              id="BookISBN"
+              placeholder="Enter ISBN"
+              type="text"
+              variant="outlined"
+            />
+            <InputLabel InputLabel="Title"></InputLabel>
+            <Input
+              height="44px"
+              width="100%"
+              colorHover="#394c73"
```
Comment (wmartinezh) [inline src/pages/books/RegisterBook.jsx]: MUI doesn't have `colorHover` check the doc [Inputs](https://mui.com/material-ui/api/input/)

**Review by wmartzh — APPROVED** [review]

**Pull request merged**